src/templates/en/accessibility_statement.html

```diff
@@ -66,6 +66,10 @@ <h2 id="conformance-status"><a href="#conformance-status" class="anchor-link">Co
           We also use Google Sheets for some interactive visualizations, which may also not be conformant to WCAG accessibility standards. These are loaded via <code>iframes</code> and we do provide images to for all our figures, and only display interactive visualizations when the client software supports this. If using a web browser and you do not wish to view the interactive visualizations, then the images can be enforce by adding a <code>?print</code> query param to any of the chapters, <a href="./2020/accessibility?print">view the Accessibility chapter in print mode</a> for one such example. We can look to make this functionality easier to activate if necessary, but all visualizations have detailed text descriptions, so we do not believe this is necessary. Please <a hreflang="en" href="https://github.com/HTTPArchive/almanac.httparchive.org/issues">let us know</a> if we are incorrect about this.
         </p>
 
+        <p>
+          We recognize that some of the color choices for our visualizations do not meet WCAG color contrast requirements. We make a conscious effort to use the more contrasting colors and labels to reduce the impact of this. However, we also believe that with the detailed descriptions provided for each visualization, plus access to the underlying data itself, the takeaways from all the visualizations are still accessible to many. We aim to improve the accessibility of our visualization color schemes in in future years.
```

Comment:
>However, we also believe that with the detailed descriptions provided for each visualization, plus access to the underlying data itself, the takeaways from all the visualizations are still accessible to many.

This feels a little defensive to me. Maybe something like, "We hope the detailed descriptions, as well as access to the underlying data itself can help with this issue until we can correct this issue."? I worry that it could be interpreted that we feel delegating the work of parsing the description/data is equal to viewing a graph, which I don't think is true.

Reply:
Thanks @ericwbailey . I went with the shorter: "We hope the detailed descriptions, as well as access to the underlying data itself can help with this issue." as felt "until we can correct this issue" repeated the sentiment of the next line and suggested the descriptions and access might go once this is resolved, when we plan to keep them.

The full addition now reads as below:

> We recognize that some of the color choices for our visualizations do not meet WCAG color contrast requirements. We make a conscious effort to use the more contrasting colors and labels to reduce the impact of this. We hope the detailed descriptions, as well as access to the underlying data itself, can help with this issue until we can correct this. We aim to improve the accessibility of our visualization color schemes in in future years.